distributed/scheduler.py

```diff
@@ -5388,7 +5388,7 @@ def report(self, msg: dict, ts: TaskState = None, client: str = None):
             try:
                 c.send(msg)
                 # logger.debug("Scheduler sends message to client %s", msg)
-            except CommClosedError:
+            except CommClosedError:  # pragma: no cover
```

Comment:
This is connected to failing batched sends which is smoething we do not have a proper solution for. There are a bunch of issues open and having this tested is important (even though we don't have a nice solution for it, yet)

```suggestion
            except CommClosedError:
```